snapcraft/cli/remote.py

```diff
@@ -159,7 +159,13 @@ def remote_build(
         branch = "master"
 
         # Send local data to the remote repository
+        if not click.confirm(
```

Comment:
why is this in this PR?

Reply:
It shouldn't, of course. Mixed branches in the hurry just before the presentation. Fixing that.